### PR TITLE
Added Documentation Tutorial

### PR DESCRIPTION
**Pull request by KitVB**
## Description

Added tutorial for creating Documentation

## Type of change

Please check the option that is related to your PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
  - In this case, we recommend to discuss your modification on GitHub issues before creating the PR
- [x] Documentations (modification for documents)

## Checklist

- [x] My code follows [the style guidelines of this project](https://deepchem.readthedocs.io/en/latest/development_guide/coding.html)
  - [x] Run `yapf -i <modified file>` and check no errors (**yapf version must be  0.32.0**)
  - [x] Run `mypy -p deepchem` and check no errors
  - [x] Run `flake8 <modified file> --count` and check no errors
  - [x] Run `python -m doctest <modified file>` and check no errors
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by KitVB** [comment]
a2
a1
Attached screenshots for reference

File: deepchem/models/keras_model.py [hunk 1]
```diff
@@ -353,7 +353,8 @@ def fit(self,
 
         Returns
         -------
-        The average loss over the most recent checkpoint interval
+        float
```
Comment (rbharath) [inline deepchem/models/keras_model.py:356]: Let's remove these changes from this PR

File: docs/source/get_started/Documentation-tutorial.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+If you want to confirm logs in more detail, use the following command:
+
+.. code-block:: shell
+
+   $ make clean html SPHINXOPTS=-vvv
+
+If you want to confirm the example tests, run:
+
+.. code-block:: shell
+
```
Comment (rbharath) [inline docs/source/get_started/Documentation-tutorial.rst:31]: Add a section about how to test whether a change to the docs renders correctly

**Review by rbharath — COMMENTED** [review]
A few more changes but getting closer

File: deepchem/models/keras_model.py [hunk 3]
```diff
@@ -738,8 +738,10 @@ def predict_on_generator(
             If specified, all outputs of this type will be retrieved
             from the model. If output_types is specified, outputs must
             be None.
-        Returns:
-            a NumPy array of the model produces a single output, or a list of arrays
+    
+        Returns
```
Comment (rbharath) [inline deepchem/models/keras_model.py:742]: Let's remove this change from this PR

File: docs/source/get_started/Documentation-tutorial.rst [hunk 4]
```diff
@@ -0,0 +1,30 @@
+Documentation Tutorial
+======================
+
```
Comment (rbharath) [inline docs/source/get_started/Documentation-tutorial.rst:3]: Add a paragraph here explaining the purpose of this page in the docs

File: docs/source/get_started/Documentation-tutorial.rst [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+.. code-block:: shell
+
+   $ pip install -r requirements.txt
+   $ make html
+   # Clean build
+   $ make clean html
+   $ open build/html/index.html
+
+To check if the changes to the docs rendered properly, open build/html on a web browser
```
Comment (rbharath) [inline docs/source/get_started/Documentation-tutorial.rst:19]: Make this a subsection about testing explicitly

**Pull request closed without merging**